help_test.go

```diff
@@ -1360,7 +1360,7 @@ DESCRIPTION:
 
 OPTIONS:
    --help, -h  show help (default: false)
-`
+   `
```

Comment:
Can we get rid of the three spaces a the last line?

Reply:
Yes. One step at a time.

Reply:
Okay fixed. Please look at the template changes.